pkg/store/mock.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // interface guard ensures Mock implements giga.PaymentsStore
-var _ giga.Store = Mock{}
+//var _ giga.Store = Mock{}
```

Comment:
No longer need the guard?

Reply:
ah! well I didn't update the mock, then I thought about deleting it all together because sqlite is in-memory and the mock was only for testing.

Reply:
er. as in we can just use sqlite for testing.